source/target/freescale/rapid_iot/target_reset.c

```diff
@@ -85,7 +92,7 @@ uint8_t target_unlock_sequence(void)
     }
 
     // verify the result
-    if (val != MDM_ID) {
+    if (val != mdm_id) {
```

Comment:
Since this code is the same for both cases you could check for either. What are you thoughts on something like this:
```C
if ((val != MDM_ID_K64)  && (val != MDM_ID_KW4 )) {
```
What you have is good as well, so don't feel like you need to change it.